packages/fxa-payments-server/public/index.html

```diff
@@ -17,7 +17,6 @@
 <body>
     <div id=fox-logo></div>
     <div id=stage>
-        <div id="top-bar"></div>
```

Comment:
Removing this so that the new Portal component can add & remove this node dynamically.